MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,471 @@
+---
+layout: post
+title: Getting Started with Maui DataForm control | Syncfusion
+description: Learn here about getting started with Syncfusion Maui DataForm (SfDataForm) control, its elements and more.
+platform: Maui
+control: SfDataForm
+documentation: ug
+---
+
+# Getting Started with Maui DataForm (SfDataForm)
+
+This section provides a quick overview of how to get started with the .NET Maui DataForm(SfDataForm) for Maui and also provides a walk-through to configure the .NET MAUI DataForm control in a real-time scenario.
```

Comment:
in all places ensure .NET Maui as .NET MAUI

Reply:
In all place .Net Maui changed to .NET MAUI